CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### 9.1.0: February, 2021
+* Update Webpack v3 to v5
+* Update Node dependencies
+* Minor change: Paths (as `@import`s) in assets are now resolved relative to `resources/` directory,
+                instead to the entrypoint stylesheet (main.scss).
```

Comment:
Would it be worth trying to restore the previous path resolving behavior? Not sure if this had been a bug with the earlier webpack components.

Reply:
The new behavior makes much more sense, and is simpler, but if we want to avoid this being a breaking change, yeah, restoring the previous behavior is needed.